crypto_kem/lightsaber/clean/kem.c

```diff
@@ -0,0 +1,79 @@
+#include "SABER_indcpa.h"
+#include "SABER_params.h"
+#include "fips202.h"
+#include "randombytes.h"
+#include "verify.h"
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+int PQCLEAN_LIGHTSABER_CLEAN_crypto_kem_keypair(unsigned char *pk, unsigned char *sk) {
+    int i;
+
+    PQCLEAN_LIGHTSABER_CLEAN_indcpa_kem_keypair(pk, sk);                         // sk[0:SABER_INDCPA_SECRETKEYBYTES-1] <-- sk
+    for (i = 0; i < SABER_INDCPA_PUBLICKEYBYTES; i++) {
+        sk[i + SABER_INDCPA_SECRETKEYBYTES] = pk[i];    // sk[SABER_INDCPA_SECRETKEYBYTES:SABER_INDCPA_SECRETKEYBYTES+SABER_INDCPA_SECRETKEYBYTES-1] <-- pk
```

Comment:
Would it not be better to move these comments at the end of the line (which are drifting anywhere to the right) to above the line?